### ISSUE TITLE
Touchpad zooming issues

### ISSUE THREAD

**Issue by Vozf** [computationalpathologygroup/ASAP#171]

**Comment by GeertLitjens** [comment]
Yes, the zooming support for touchpad and touchscreen gestures is not really implemented. It depends how the operating systems translates these gestures currently. It is on my to-do list to add better support.

**Comment by ntomita** [comment]
@GeertLitjens Do you have a list of keyboard shortcut I can use on ASAP, including zooming out. I tried to find documentation but couldn't find one. Thanks!
